conjure-java-core/src/main/java/com/palantir/conjure/java/ConjureTags.java

```diff
@@ -44,7 +44,7 @@ public static boolean isUnsafe(Collection<String> tags) {
         return tags.contains(UNSAFE);
     }
 
-    public static ImmutableList<AnnotationSpec> tagAnnotations(ArgumentDefinition argument) {
+    public static ImmutableList<AnnotationSpec> safetyAnnotations(ArgumentDefinition argument) {
```

Comment:
I assume dependency on the Safe class is automatically inherited in generated code?

Reply:
That's correct -- generated code already uses safe-logging preconditions which provides the safe-logging annotation API as an api dependency.